egs/iwslt18/st1/conf/tuning/train_rnn_char.yaml

```diff
@@ -30,6 +30,7 @@ weight-decay: 1e-6
 # optimization related
 sortagrad: 0 # Feed samples from shortest to longest ; -1: enabled for all epochs, 0: disabled, other: enabled for 'other' epochs
 opt: adadelta
+eps-decay: 0.01
```

Comment:
This is interesting. Someone reported some improvement by making `eps-decay` larger. Did you also observe the improvement?

Reply:
Yes. eps-decay: 0.01 is a little bit large for BPE models.